resources/lang/en/registration.php

```diff
@@ -8,12 +8,12 @@
   'email_success' => '<strong>:email</strong> successfully verified',
   'hi' => 'Hi :first_name!',
   'login' => 'Login',
-  'one_step' => 'You are now one step to owning a leading school management system.',
-  'thanks' => 'thanks for signing up, login below to begin setting up <strong>:name</strong>',
+  'one_step' => 'I congratulate you on taking this bold step of owning a Learning School Management System.',
+  'thanks' => 'thanks for signing up, we are so excited to have you commence this audacious journey with us! login below to begin setting up <strong>:name</strong>',
   'token_expired' => 'Token expired',
   'invited' => 'You have been invited by <strong>:school_owner</strong>, to join <strong>:school_name</strong>',
-  'confirm' => 'confirm your email below to begin setting up your profile',
-  'welcome' => 'Welcome to :name',
+  'confirm' => 'To get you started please kindly confirm your email address below to begin setting up your profile',
+  'welcome' => 'Welcome to :Nero',
```

Comment:
```suggestion
  'welcome' => 'Welcome to :name',
```
`:name` is passed in as an argument to this file, see [here](https://github.com/purplenimbus/edu-api/blob/master/app/Notifications/ActivateTenant.php#L41) for more into and ultimately , `Nero` will replace :name